### ISSUE TITLE
A feature-rich app like this requires a proper documentation

### ISSUE THREAD

**Issue by sharu725** [jhlywa/chess.js#466]
I think README is not good enough to demonstate some of the amazing features this app has to offer.

The README also lacks integration with chess boards like chessground.

It can be a lot useful if there was a dedicated documentation page with live demo, code sample, etc.

**Comment by neofight78** [comment]
I don't think we can provide detailed instructions on integrating with various other libraries. Perhaps address those projects with your requests, if they explicitly support chess.js. If there is anything specific about the chess.js library itself that you are having problems with, then feel free to raise an issue and we may be able to help.
